src/component/handlers/edit/editOnCopy.js

```diff
@@ -31,7 +33,31 @@ function editOnCopy(editor: DraftEditor, e: SyntheticClipboardEvent<>): void {
     return;
   }
 
-  editor.setClipboard(getFragmentFromSelection(editor._latestEditorState));
+  const fragment = getFragmentFromSelection(editor._latestEditorState);
+
+  editor.setClipboard(fragment);
+
+  // IE11 does not support ClipboardEvent.clipboardData.
+  if (e.clipboardData && fragment) {
+    const content = ContentState.createFromBlockArray(fragment.toArray());
+    const serialisedContent = JSON.stringify(
+      convertFromDraftStateToRaw(content),
+    );
+
+    const fragmentElt = document.createElement('div');
+    const domSelection = window.getSelection();
+    fragmentElt.appendChild(domSelection.getRangeAt(0).cloneContents());
```

Comment:
This is inspired from Rangy: https://github.com/timdown/rangy/blob/1e55169d2e4d1d9458c2a87119addf47a8265276/src/core/domrange.js#L515-L520.

Also from what I understand modern browsers only support one range in the selection.